requirements.txt

```diff
@@ -1,6 +1,10 @@
-opencv-python==4.1.0.25
+#opencv-python==4.1.0.25 #(could not install)
```

Comment:
Will need to double check that this file works or we should remake this?

Reply:
This could be more general of it was 
```
opencv-python>=4.1,<4.2
```
(On my phone, haven't tested this)